runtime/doc/lua.txt

```diff
@@ -4308,4 +4308,30 @@ vim.text.hexencode({str})                               *vim.text.hexencode()*
     Return: ~
         (`string`) Hex encoded string
 
+
+==============================================================================
+Lua module: tohtml                                                    *tohtml*
+
+
+:TOhtml {file}                                                       *:TOhtml*
+Generates HTML and writes to a file and opens it in a new split window. If the
+file is not given then it uses a temporary file.
+
+tohtml({winid}, {opt})                                       *tohtml.tohtml()*
+    Converts a window to HTML and returns the output as a list of string.
```

Comment:
```suggestion
    Converts the buffer shown in the window {winid} to HTML and returns the output as a list of string.
```
This is slightly ambiguous, since it could also be interpreted as converting only what is _shown_ in the window.